frontend/lib/src/components/widgets/AudioInput/AudioInput.tsx

```diff
@@ -72,6 +68,8 @@ import {
 } from "./constants"
 import formatTime from "./formatTime"
 import AudioInputActionButtons from "./AudioInputActionButtons"
+import convertMp4ToWav from "./convertAudioToWav"
```

Comment:
nit: I think this is still using the old name here `convertMp4ToWav` -> `convertAudioToWav`